### DISCUSSION TITLE
【渲染 Onedrive】关于增加 onedrive 预览

### DISCUSSION THREAD

**Discussion by wxyhgk** [tangly1024/NotionNext#2010] [Ideas]
下面的图中可以看到,在 notion 中可以直接显示,并且很好

<img width="741" alt="image" src="https://user-images.githubusercontent.com/37659181/201472798-7556352c-e9d3-410f-b7b4-c3faabf31e5c.png">

在网页端就显示不好了
<img width="543" alt="image" src="https://user-images.githubusercontent.com/37659181/201472832-797be0c5-665a-4cf2-b69e-d5862ffc8d05.png">

onedrive 是可以直接分享,不需要登录的网盘,对于分享资料什么的很好,所以我希望增加这个,谢谢.

**Comment by tangly1024** [comment]
已收到反馈，我需要先研究一下onedrive😂

**Comment by lifeafter619** [comment]
> 已收到反馈，我需要先研究一下onedrive😂

好哇，大佬应该整个Onedrive e5开发订阅的

**Comment by lifeafter619** [comment]
Onedrive的business版甚至支持嵌入视频到notion内了（Microsoft stream），就是要登录组织内有权限的账户，不太友好
